.github/workflows/ci.yml

```
@@ -28,6 +28,8 @@ jobs:
         os: [ubuntu-latest, windows-latest, macOS-latest]
     runs-on: ${{ matrix.os }}
     steps:
+      - if: matrix.os == 'ubuntu-latest'
+        run: sudo apt install libwayland-cursor0 libxkbcommon-dev
```

```suggestion
        run: sudo apt install libwayland-cursor0 libxkbcommon-dev libwayland-dev
```

Should fix CI once and for all.